chucks-pick3-web/angular-app/src/app/components/thank-you.component/thank-you.component.ts

```diff
@@ -0,0 +1,16 @@
+import {Component, OnInit} from '@angular/core';
+
+@Component({
+  selector: 'app-thank-you',
+  templateUrl: './thank-you.component.html',
+  styleUrls: ['./thank-you.component.css']
+})
+export class ThankYouComponent implements OnInit {
```

Comment:
Lets rename this component to SignUpConfirmationComponent.  The directory and file name should be: 

1. `src/app/components/sign-up-confirmation.component/sign-up-confirmation.component.ts`
2. `src/app/components/sign-up-confirmation.component/sign-up-confirmation.component.html`
3. `src/app/components/sign-up-confirmation.component/sign-up-confirmation.component.css`

Reply:
Completed changes for naming convention of sign up component.